analytics.go

```diff
@@ -74,10 +74,7 @@ func nestedGet(key string, data interface{}) (value interface{}) {
 		switch t := value.(type) {
 		case map[string]interface{}:
 			value = t[k]
-
-		default:
-			// value doesn't exist b/c parent value isn't a map
-			panic(fmt.Sprintf("err mer gerrrd, %s doesn't exist in %s", k, key))
+			// ignore any other types/cases
```

Comment:
I may be wrong about this, but maybe want to test this theory... 

Consider returning nil isntead of ignoring. Otherwise you may end up returning the parent value, or attempting to re-check the parent value for the next nested value

consider example blob:
```
{
  "value1": {
    "value2": {
      "value3: "someString"
    }
  }
}
```
If you're looking for `value1.value4` - it may return `value1`
if you're looking for `value1.value3` - it may still pull `value3`, which would be incorrect